src/app/actions/sendEmail.ts

```
@@ -1019,331 +288,101 @@ Para esclarecimentos, entre em contato: ${rejectedData.contactEmail}
-</body>
-</html>
-        `
+        html: generateEmailHtml('registration-rejected', rejectedData)
       };
 
     case 'payment_reminder':
-      const paymentData = emailData.data;
+      const reminderData = emailData.data;
       return {
-        subject: `💰 Lembrete de Pagamento - ${paymentData.assemblyName}`,
+        subject: `💳 Lembrete de Pagamento - ${reminderData.assemblyName}`,
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)